legal-api/src/legal_api/services/search_service.py

```diff
@@ -269,3 +269,30 @@ def get_search_filtered_filings_results(business_json,
                 draft_results.append(draft)
 
         return draft_results
+
+    @staticmethod
+    def get_affiliation_mapping_results():
+        """Return affiliation mapping results for the given organization ID."""
+        query = db.session.query(
+            Filing.id.label('filing_id'),
+            Business.identifier,
+            Filing
+            .filing_json['filing'][Filing._filing_type]['nameRequest']['nrNumber']  # pylint: disable=protected-access
+            .label('nrNum'),
+            RegistrationBootstrap.identifier.label('temp_reg')  # pylint: disable=no-member
+                ).select_from(Filing) \
+            .outerjoin(Business, Filing.business_id == Business.id) \
+            .join(RegistrationBootstrap, Filing.temp_reg == RegistrationBootstrap.identifier)
+
+        rows = query.all()
+
+        result_list = [
+            {
+                'filingId': row.filing_id,
+                'identifier': row.identifier,
+                'bootstrapIdentifier': row.temp_reg,
+                'nrNumber': row.nrNum
+            }
+            for row in rows
+        ]
```

Comment:
Probably possible to just use labels that match your dict? not sure if there's a mechanism for LEAR to change a query to an dict, maybe there's a to_dict laying around somewhere?